Makefile

```diff
@@ -5,3 +5,7 @@ test:
 .PHONY: build
 build:
 	go build ./...
+
+.PHONY: clean
+clean:
+	go clean ./...
```

Comment:
These are in https://github.com/go-piv/piv-go/pull/139, and should be dropped from this PR in cleanup